src/main/java/core/basesyntax/AreaCounter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCounter {
```

Comment:
```suggestion
public interface AreaCalculator {
```

i think Calculator is more suitable name